JS/app.js

```diff
@@ -0,0 +1,214 @@
+
+for (let i = 2; i<=200; i+=2) {
+	if(i % 2 === 0){	
+	console.log(i);
+}
+}
+//----------------Excited Kitten----------------//
+//1.Write code that logs "Loves me, pet me! HSSSSSSS!" 20 times
+//2.For every even number in your loop, log "...human...why you taking pictures of me?...",
+// "...the catnip made me do it...", or "...why does the red dot always get away..." at random.
+
+
+
+//created variables that hold the strings
+const catOne = "...human...why you taking pictures of me?...";
+const catTwo = "...the catnip made me do it...";
+const catThree = "...why does the red dot always get away...";
+
+
+//loop that logs "Love me, pet me! HSSSSSS!" 20 times
+for (let i = 0; i <= 20; i++){
+			console.log("Love me, pet me! HSSSSSS!");
+//check to see if the number that i is going through
+	if(i % 2 === 0){
+		//made variable that holds the function Math.random
+		let randCat = Math.random();
+		//if the random number generated is less than equal to .33, log catOne
+	  if (randCat <= .33){
+	  	console.log(catOne);
+	  			//if the random number generated is less than equal to .66, log catTwo
+	} else if (randCat <= .66) {
+		console.log(catTwo);
+		//any other number above .66, log catThree
+	} else {
+		console.log(catThree);
+	}
+	}
+}
+
+
+
+// Write a javascript application that logs all numbers from 1 - 100.
+// If a number is divisible by 3 log "Fizz" instead of the number.
+// If a number is divisible by 5 log "Buzz" instead of the number.
+// If a number is divisible by 3 and 5 log "FizzBuzz" instead of the number.
+
+// setting up loop
+
+for (let i = 1; i < 100; i++) {
+	//if i is divisible by 3 and 5, console log "fizzbuzz" instead of the number
+	if (i % 3 && 5 === 0)
+		console.log("FizzBuzz");
+	//otherwise if i is divisible by 3 console log "fizz"
+	else if(i % 3 === 0)
+		console.log("fizz");
+	//otherwise if i is divisble by 5 console log "buzz"
+	else if(i % 5 === 0)
+		console.log("buzz");
+	//console logging 0-100
+	else console.log(i);	
+}
+
+// ---------Getting to know you --------- //
+// Use the following arrays to answer the questions below (name, age, hometown):
+const shahzad = ["Shahzad", 1000, "Austin"];
+const jim = ["Jim", 16, "All cities"];
+const reuben = ["Reuben", 22, "Durham"];
+const daniel = ["Daniel", 186, "Dallas"];
+const ryan = ["Ryan", 65, "Denver"];
+
+// Daniel decides that Shahzad can't be named "Shahzad" anymore. Remove "Shahzad" from the shahzad array and replace it with "Gameboy".
+
+//use .splice to take out shahzad and replace it with Gameboy
+shahzad.splice(0, 1, 'Gameboy');
+console.log(shahzad);
+
+// //Daniel just had his birthday; change Daniel's array to reflect him being a year older. Don't just hard code 187--pretend that you don't know that his age is 186, and write your code to just make him a year older than whatever age he was.
+
+//use .splice to replace daniel's age with 19
+daniel.splice(1,1, 19);
+console.log(daniel);
```

Comment:
no need for splice.  in the previous problem, ok, because it said "remove and then put something else in." but here it just said change the value.  also it said don't hard code the new value.  also the original value was 186.  so the answer we were looking for is a little less complex:
`daniel[1] = daniel[1] + 1`.
or
`daniel[1]++`